app/models/classified_listing.rb

```diff
@@ -67,10 +67,10 @@ def self.categories_available
       "education" => { cost: 1, name: "Education/Courses", rules: "Educational material and/or schools/bootcamps." },
       "jobs" => { cost: 25, name: "Job Listings", rules: "Companies offering employment right now." },
       "mentors" => { cost: 1, name: "Offering Mentorship", rules: "You are available to mentor someone." },
-      "products" => { cost: 10, name: "Products/Tools", rules: "Must be available right now." },
+      "products" => { cost: 5, name: "Products/Tools", rules: "Must be available right now." },
```

Comment:
Accept this suggestion and dismiss the other suggestion if it's supposed to cost **`10`**.
```suggestion
      "products" => { cost: 10, name: "Products/Tools", rules: "Must be available right now." },
```